karoo_gp/tree.py

```diff
@@ -24,9 +23,9 @@ def load(cls, id, expr, tree_type='f'):
             tree_type = expr[0]
             expr = expr[1:]
         elif expr[0] != '(':
-            raise ValueError('Load-from expressions must start with tree type'
-                             '("f"/"g") or "(".')
-        root = Branch.load(expr, tree_type)
+            raise ValueError(f'Load-from expressions must start with tree type'
+                             f' (f/g) or parenthesis; got {expr[0]}')
```

Comment:
```suggestion
            raise ValueError(f'Load-from expressions must start with tree type '
                             f'(f/g) or parenthesis; got {expr[0]!r}')
```

Reply:
`expr` expected be a string, and I thought fstrings kinda call `repr` automatically?

Reply:
```pycon
>>> s = "a string!"
>>> f'Expected an int, got {s}'
'Expected an int, got a string!'
>>> f'Expected an int, got {s!r}'
"Expected an int, got 'a string!'"
```